pkg/codegen/main.go

```diff
@@ -19,7 +19,7 @@ import (
 
 func main() {
 	controllergen.Run(args.Options{
-		ImportPackage: "github.com/rancher/wrangler/v2/pkg/generated",
+		ImportPackage: "github.com/rancher/wrangler/v3/pkg/generated",
 		OutputPackage: "github.com/rancher/wrangler/pkg/generated",
```

Comment:
Does this need to be updated?

Reply:
No it's not needed. Output package ends up being the path where the generated files are written to. For example: `/tmp/gopath/src/github.com/wrangler/pkg/generated`. Then whatever is under `/tmp/gopath/src/github.com/rancher/wrangler` is copied back to the current working directory. If that path had `v2/` in it, it would mess up the destination of the generated files.

That's why we both have ImportPackage, which does contain the v2 suffix, and OutputPackage, which doesn't.